nims-notes-things/src/components/notes.js

```diff
@@ -13,9 +14,12 @@ function Notes() {
         return (
           <div>
             <Note>
-              <p>{n.name}</p>
-              <p>{n.date}</p>
-              <p>{n.note}</p>
+              <img src={Pin} height={20} alt="" />
```

Comment:
you can use a styled component, or for more of a challenge create a custom image component.